module-bsp/board/rt1051/bluetooth/BlueKitchen.cpp

```diff
@@ -17,62 +15,57 @@ using namespace bsp;
 
 BlueKitchen::BlueKitchen()
 {
-    read_buff     = NULL;
-    read_ready_cb = NULL;
-    write_done_cb = NULL;
+    readBuffer        = nullptr;
+    readReadyCallback = nullptr;
+    writeDoneCallback = nullptr;
 }
 
 BlueKitchen::~BlueKitchen() = default;
 
-BlueKitchen *BlueKitchen::getInstance()
+BlueKitchen &BlueKitchen::getInstance()
 {
-    static BlueKitchen *k = NULL;
-    if (k == NULL) {
-        /// outcomming & incomming heap allocated buffers sizes
-        /// packet on IP network cna have MTU 1500, so big enough buffers were added to not throttle comms
-        k = new BlueKitchen();
+    static std::unique_ptr<BlueKitchen> instance;
+    if (instance == nullptr) {
+        instance = std::make_unique<BlueKitchen>();
     }
-    return k;
+    return *instance;
 }
 
-BTdev::Error BlueKitchen::read(uint8_t *buf, size_t nbytes)
+BTDevice::Error BlueKitchen::read(uint8_t *buf, size_t nbytes)
```

Comment:
To be consistent with previous changes. I'd add `std::` to `uint8_t` and `size_t`. I'd changed `nbytes` to `size`. However, feel free to resolve it if it's not important ;)

Reply:
You're right, I tried to change it everywhere but I must've missed that one, thanks!